client/components/Map.tsx

```diff
@@ -94,9 +90,8 @@ const styles = StyleSheet.create({
       height: '100%',
     },
     listButton: {
-        position: "absolute", 
-        top: 80,
-        right: 30,
+        position: 'absolute', 
+        top: getStatusBarHeight() + 10,
```

Comment:
cool

Reply:
I was having a devil of a time trying to get things to not interfere with the status bar but I'm 90% sure that library and function will work for both iOS and Android.